docs/zkapps/tutorials/01-hello-world.mdx

```diff
@@ -63,8 +63,19 @@ Now that you have the tooling installed, we can start building our application.
 
 First, create a new project using:
 
+<!--
+
+Configure `git` email and name, otherwise `zkapp-cli` project generation will return a non-zero exit code.
+
+```sh
+$ git config --global user.email "test@example.com"
+$ git config --global user.name "Test"
+```
+
+-->
+
 ```sh
-zk project 01-hello-world
+$ zk project 01-hello-world
```

Comment:
@rjdellecese Is it possible to make this work without adding `$`? 

Removing this: 1.) cleans it up, particularly for newbies, b/c there's one less thing to visually parse, 2.) avoids getting copied--though we could update the JS copy code to strip it. 

Up to this point, the pattern we've followed for tutorials is to avoid using `$` unless the block contains _output_ from the command(s) as well. 

If it is required for you, then we can update the JS copy snippet and accept the added visual noise.

Reply:
Yeah, I don't think that would be too hard. I personally like it for consistency's sake, though—in my experience inconsistency can be the most confusing thing for new learners, because when you lack context/prior knowledge, you generally start by looking for the simplest patterns you can find (such as "every shell command that I need to execute is preceded by a `$ `, rather than "sometimes `$ ` prepends a shell command that I need to execute, but sometimes it doesn't").

Another consideration is that if we updated the JS copy code to seek out lines prepended with `$ ` and then strip the `$ ` then it would also work with code blocks like those that currently intermix commands and output, which would be nice.

I'm still happy to change it if we'd prefer, just want to share a competing view! Also thanks for pointing out that this would break copying from certain shell code blocks, I had overlooked that.

Reply:
> Another consideration is that if we updated the JS copy code to seek out lines prepended with $ and then strip the $ then it would also work with code blocks like those that currently intermix commands and output, which would be nice.
This is a great idea, and one we'll do. I'll create an issue because we can do this no matter which approach we take for single line commands. 

I'm still kicking around this idea. Your consistency argument is a fair point. An additional is that for any noobies who use their mouse to select/copy/paste, having only the command as selectable text is better for usability in such case.

Reply:
> An additional is that for any noobies who use their mouse to select/copy/paste, having only the command as selectable text is better for usability in such case.

Very good point as well.

Reply:
@rjdellecese Let's go with `$` for all, single and multi-line shell commands please. Either way will work, but just wanting to make a decision so this can move forward. We'll need to queue this PR until a corresponding PR is ready to update all tutorials to add the `$` to the code.
CC @ymekuria for that since he's working on the JS-snippet related code.

Reply:
Update: All tutorials [have been updated](https://github.com/o1-labs/docs2/pull/247) to add `$` and the JS code copy snippet has been updated to strip it and any console output as well.